spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.feature 'Movie Show', type: :feature do
+  
+
+  before(:each) do
+    @studio_1 = Studio.create(name: "Sleepy Time Movies", location: "OKC")
+    @studio_2 = Studio.create(name: "Show Biz Productions", location: "LA")
+    @movie_1 = @studio_1.movies.create(title: "Blaine's Big Day", creation_year: '2023', genre: 'Tragedy')
+    @movie_2 = @studio_2.movies.create(title: 'Sloppy Code', creation_year: '2023', genre: 'Horror')
+    @actor_1 = @movie_1.actors.create(name: "Blaine Kennedy", age: 30)
+    @actor_2 = @movie_1.actors.create(name: "Colin Kennedy", age: 34)
+    @actor_3 = Actor.create(name: "Patty Kennedy", age: 65)
+  end
+
+
+  describe "I visit a movie's show page." do
+    it "I see the movie's title, creation year, and genre, and a list of all its actors from youngest to oldest and I see the average age of all of the movie's actors" do 
+     
+
+      visit "/movies/#{@movie_1.id}"
+      
+
+      expect(page).to have_content(@movie_1.title) 
+      expect(page).to have_content(@movie_1.creation_year) 
+      expect(page).to have_content(@movie_1.genre) 
+      expect(page).to have_content(@actor_1.name) 
+      expect(page).to have_content(@actor_1.age)
+      expect(page).to have_content(@actor_2.name) 
+      expect(page).to have_content(@actor_2.age) 
+      expect(page).to have_content(32) 
+    end
+  end
```

Comment:
These tests are ok but I think they could be more verbose, for example right now we are just checking that 32 shows up on the page somewhere. I think it would be much better to have an assertion like this:

`expect(page).to have_content("Average Age: 32") `

This way we reduce the ambiguity of what 32 represents on the page since 32 could be a runtime, and age, the time of day, etc... previously your tests only says does 32 show up anywhere on the page. This could produce a false positive.